app/assets/stylesheets/provider/_theme.scss

```diff
@@ -1,11 +1,3 @@
-// compass css basics
-@import "compass/utilities/general/clearfix";
-@import "compass/typography/lists/inline-block-list";
-@import "compass/css3";
-@import "compass/reset/utilities";
-@import "compass/typography/text/ellipsis";
-@include reset-html5;
```

Comment:
This is supposed to reset some things, but I haven't seen anything weird in the admin portal.

Reply:
This looks like a legacy feature from times when HTML5 was the novelty. Also, it just sets `display: block;` to everything, so not very relevant 😂.